grammars/php.cson

```diff
@@ -2203,7 +2405,13 @@
           '4':
             'name': 'punctuation.definition.variable.php'
         'comment': 'Simple syntax with braces: "foo${bar}baz"'
-        'match': '(?x)\n\t\t\t\t\t\t((\\$\\{)(?<name>[a-zA-Z_\\x{7f}-\\x{ff}][a-zA-Z0-9_\\x{7f}-\\x{ff}]*)(\\}))\n\t\t\t\t\t\t'
+        'match': '''
+          (?x)
+          (
+            (\\$\\{)
```

Comment:
It for easy read/recognize. When it ass \\{ it become escaped character so easy to eye catch and see so it symbol not repetition.